src/main/java/com/synopsys/integration/alert/web/audit/AuditEntryActions.java

```diff
@@ -113,8 +113,31 @@ public AuditEntryModel get(final Long id) {
         return null;
     }
 
-    public AlertPagedModel<AuditEntryModel> resendNotification(final Long notificationdId, final Long commonConfigId) throws IntegrationException {
-        final NotificationContent notificationContent = notificationManager.findById(notificationdId).orElseThrow(() -> new AlertNotificationPurgedException("No notification with this id exists."));
+    public JobAuditModel getAuditInfoForJob(final Long jobId) {
+        if (jobId != null) {
+            Optional<AuditEntryEntity> optionalAuditEntryEntity = auditEntryRepository.findFirstByCommonConfigIdOrderByTimeLastSentDesc(jobId);
+            if (optionalAuditEntryEntity.isPresent()) {
+                AuditEntryEntity auditEntryEntity = optionalAuditEntryEntity.get();
+                String timeCreated = null;
+                if (null != auditEntryEntity.getTimeCreated()) {
+                    timeCreated = notificationContentConverter.getContentConverter().getStringValue(auditEntryEntity.getTimeCreated());
+                }
+                String timeLastSent = null;
+                if (null != auditEntryEntity.getTimeLastSent()) {
+                    timeLastSent = notificationContentConverter.getContentConverter().getStringValue(auditEntryEntity.getTimeLastSent());
+                }
+                String status = null;
+                if (null != auditEntryEntity.getStatus()) {
+                    status = auditEntryEntity.getStatus();
+                }
+                return new JobAuditModel(timeCreated, timeLastSent, status);
+            }
+        }
+        return null;
```

Comment:
If we're returning null here should this method return an Optional and return an Optional.empty().  The controller would need to change to return an Optional<JobAuditModel> but what I have initially researched is that the controller returning an Optional.empty will return a 404 that the resource cannot be found which is valid here given that the jobId is a path variable indicating a specific resource.

Reply:
Would need to test the Optional returned from the controller to be sure it returns a 404